assets/base.css

```diff
@@ -2679,6 +2679,7 @@ details[open] > .header__submenu {
 }
 
 .header__menu-item {
+  font-size: 1.4rem;
```

Comment:
It's already receiving this font size from the `.link` class, so I don't think we need to re add it here 🤔 Though I might be missing a use case in which it is needed 😅